src/components/DpTooltip/utils/tooltip.js

```diff
@@ -5,9 +5,11 @@ import { v4 as uuid } from 'uuid'
 let handleShowTooltip = null
 let handleHideTooltip = null
 let handleTimeoutForDestroy = null
+let tooltips = []
 
 const deleteTooltip = (tooltipEl) => {
   if (tooltipEl) {
+    tooltips.filter(el => el.id !== tooltipEl.id)
```

Comment:
what for this is needed?

Reply:
refactored it https://github.com/demos-europe/demosplan-ui/pull/542/commits/f4102553846455c57f7e55a9f4174056425a5cf8